scimath/Functionals/Function1D.h

```diff
@@ -84,7 +84,6 @@ template<class T, class U=T> class Function1D : public Function<T,U> {
   explicit Function1D(const uInt n) : Function<T,U>(n) {}
   explicit Function1D(const Vector<T> &in) : Function<T,U>(in) {}
   Function1D(const FunctionParam<T> &other) : Function<T,U>(other) {}
-  Function1D(const Function1D<T,U> &other) : Function<T,U>(other) {}
```

Comment:
Why does this disappear?

Reply:
Same reason as before, this copy constructor causes warnings, the default is fine.